### PR TITLE
gnn documentation update

### PR DESCRIPTION
**Pull request by sjiang87**
- update the documentation for stack_mpnn.py
- add some data generation functions

### CODE REVIEW

File: deephyper/contrib/layers/stack_mpnn.py [hunk 1]
```diff
@@ -1,7 +1,99 @@
 import tensorflow as tf
+import numpy as np
+import tensorflow.keras.backend as K
+from numpy.random import seed
 from tensorflow.keras import activations
 from tensorflow.keras.layers import Dense
 
+seed(2020)
+tf.random.set_seed(2020)
```
Comment (Deathn0t) [inline deephyper/contrib/layers/stack_mpnn.py:9]: The random seed should never be set globally otherwise it has side-effect. If we import any object from this module the random state will be fixed. For numpy, it is better to use local `np.random.RandomState`. The seed to fix a neural network training is performed in the `run_base_trainer`: https://github.com/deephyper/deephyper/blob/develop/deephyper/nas/run/_run_base_trainer.py#L62

**Review by Deathn0t — CHANGES_REQUESTED** [review]
Please correct the global random state affectation.

File: deephyper/contrib/layers/stack_mpnn.py (hunk 1 above)
Comment (sjiang87) [reply deephyper/contrib/layers/stack_mpnn.py]: I have removed the random seeds.
